src/node.cc

```diff
@@ -3257,8 +3258,12 @@ void LoadEnvironment(Environment* env) {
 
   env->SetMethod(env->process_object(), "_rawDebug", RawDebug);
 
+  // Expose the global object as a property on itself
+  // (Allows you to set stuff on `global` from anywhere in JavaScript.)
+  global->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "global"), global);
+
   Local<Value> arg = env->process_object();
-  f->Call(global, 1, &arg);
+  f->Call(Null(env->isolate()), ARRAY_SIZE(&arg), &arg);
```

Comment:
It's up for debate whether something that works by accident is wrong or not but if you're back-porting this anyway, I'd recommend changing it to something that's obviously correct.  I'll update my back-port PR accordingly, I can simply drop the commit that fixes up this line.